src/components/member-profile/contribution-type/index.js

```diff
@@ -54,7 +65,22 @@ const ContributionType = (props) => {
       <div className={showMoreContentClass}>
         {type !== 'Active tasks' ? (
           <div>
-            {renderContributions(contributions, fullName, imageLink, devUser)}
+            {renderContributions(
+              contributions.slice(0, count),
+              fullName,
+              imageLink,
+              devUser
+            )}
+            {count <= contributions.length && devUser && (
+              <div className={classNames.loadDiv}>
+                <button
+                  className={classNames.loadButton}
+                  onClick={loadMoreHandler}
```

Comment:
Appreciate that you haven't added inline functions, a good practice to follow as inline functions are anonymous. They are created every time the component renders. 😄